app/lib/features/space/pages/pins_page.dart

```diff
@@ -19,81 +19,106 @@ class SpacePinsPage extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    final spaceRelationsOverview =
```

Comment:
that's not the right provider for this. there is a provider that directly yields the space profile with the membership. We don't need that entire relationship thing here.